src/App.js

```diff
@@ -1,17 +1,15 @@
 import React from "react";
-import cars from "./constants/car/carsInfo";
-import CardList from "./components/CardList/CardList";
-import Header from "./components/Header/Header";
-import Footer from "./components/Footer/Footer";
 import "./App.css";
+import Router from "./components/Router";
+import { BrowserRouter } from "react-router-dom";
```

Comment:
Imports order

1. React and React Hooks
2. packages
3. components
4. styles